Artesian/Artesian.SDK/Dto/Auction/AuctionBidValue.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System.ComponentModel.DataAnnotations;
+using EnsureThat;
+using MessagePack;
+using KeyAttribute = MessagePack.KeyAttribute;
+
+namespace Artesian.SDK.Dto
+{
+    /// <summary>
+    /// The Auction Data for a single Multiple timestamps
+    /// </summary>
+    [MessagePackObject]
+    public class AuctionBidValue
+    {
+        /// <summary>
+        /// The Auction Data for a single Multiple timestamps
+        /// </summary>
+        /// <param name="price"></param>
+        /// <param name="quantity"></param>
+        public AuctionBidValue(double price, double quantity)
+        {
+            EnsureArg.IsGt(quantity, 0);
```

Comment:
I'm not sure this is matched server-side nor it's required from the wiki specification. Please verify with Alessandro.